src/Facets/GlacisFacet.sol

```diff
@@ -0,0 +1,112 @@
+/// @notice Integration of the Glacis airlift (wrapper for native token bridging standards)
+/// @custom:version 1.0.0
+contract GlacisFacet is ILiFi, ReentrancyGuard, SwapperV2, Validatable {
+    /// Storage ///
+
+    /// @notice The contract address of the glacis airlift on the source chain.
+    IGlacisAirlift public immutable airlift;
+
+    /// Types ///
+
+    /// @param refund Refund address
+    /// @param nativeFee The fee amount in native token required by the Glacis Airlift.
+    struct GlacisData {
+        address refund;
+        uint256 nativeFee;
+    }
+
+    /// Constructor ///
+    /// @notice Initializes the GlacisFacet contract
+    /// @param _airlift The address of Glacis Airlift contract.
+    constructor(IGlacisAirlift _airlift) {
+        airlift = _airlift;
+    }
+
+    /// External Methods ///
+
+    /// @notice Bridges tokens via Glacis
+    /// @param _bridgeData The core information needed for bridging
+    /// @param _glacisData Data specific to Glacis
+    function startBridgeTokensViaGlacis(
+        ILiFi.BridgeData memory _bridgeData,
+        GlacisData calldata _glacisData
+    )
+        external
+        payable
+        nonReentrant
+        refundExcessNative(payable(msg.sender))
+        validateBridgeData(_bridgeData)
+        doesNotContainSourceSwaps(_bridgeData)
+        doesNotContainDestinationCalls(_bridgeData)
+    {
+        LibAsset.depositAsset(
+            _bridgeData.sendingAssetId,
+            _bridgeData.minAmount
+        );
+        _startBridge(_bridgeData, _glacisData);
+    }
+
+    /// @notice Performs a swap before bridging via Glacis
+    /// @param _bridgeData The core information needed for bridging
+    /// @param _swapData An array of swap related data for performing swaps before bridging
+    /// @param _glacisData Data specific to Glacis
+    function swapAndStartBridgeTokensViaGlacis(
+        ILiFi.BridgeData memory _bridgeData,
+        LibSwap.SwapData[] calldata _swapData,
+        GlacisData calldata _glacisData
+    )
+        external
+        payable
+        nonReentrant
+        refundExcessNative(payable(msg.sender))
+        containsSourceSwaps(_bridgeData)
+        doesNotContainDestinationCalls(_bridgeData)
+        validateBridgeData(_bridgeData)
+    {
+        _bridgeData.minAmount = _depositAndSwap(
+            _bridgeData.transactionId,
+            _bridgeData.minAmount,
+            _swapData,
+            payable(msg.sender)
+        );
+        _startBridge(_bridgeData, _glacisData);
+    }
+
+    /// Internal Methods ///
+
+    /// @dev Contains the business logic for the bridge via Glacis
+    /// @param _bridgeData The core information needed for bridging
+    /// @param _glacisData Data specific to Glacis
+    function _startBridge(
+        ILiFi.BridgeData memory _bridgeData,
+        GlacisData calldata _glacisData
+    ) internal {
+        SafeERC20.safeTransfer(
+            IERC20(_bridgeData.sendingAssetId),
```

Comment:
Even though this is simple code, I would suggest to comment at least on the main steps (one comment for the transfer, here you could explain that we are calling a function that requires the tokens to be sent to the contract already) and leave a blank line between unrelated code blocks.